bindings/profilers/wall.cc

```diff
@@ -489,8 +489,11 @@ Result WallProfiler::StartImpl() {
 }
 
 std::string WallProfiler::StartInternal() {
+  // Reuse the same names for the profiles because strings used for profile
+  // names are not released until v8::CpuProfiler object is destroyed.
+  // https://github.com/nodejs/node/blob/b53c51995380b1f8d642297d848cab6010d2909c/deps/v8/src/profiler/profile-generator.h#L516
   char buf[128];
-  snprintf(buf, sizeof(buf), "pprof-%" PRId64, profileIdx_++);
+  snprintf(buf, sizeof(buf), "pprof-%" PRId64, (profileIdx_++) % 2);
```

Comment:
I have to admit I don't understand this.

Reply:
As discussed, modulo operation allows to alternate between "pprof-0" and "pprof-1",  we know that there are most 2 profiles active at the same time.